regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -72,6 +96,10 @@ var CommentReviewView = Backbone.View.extend({
     this.render();
   },
 
+  toggleSubmit: function() {
+    $('.submit-button').prop('disabled', function(i, v) { return !v; });
```

Comment:
Good point. If we were to update it template-side, would it be inline JS or something else..?